### DISCUSSION TITLE
Probleme LS22 Oberkiorchfeld

### DISCUSSION THREAD

**Discussion by Gjardiunn** [Courseplay/Courseplay_FS22#2700] [General]
Hallo, ich bin neu bei LS22 und nutze gerne das mod courseplay. Ich habe aber massive probleme beim Kurs generieren auf der Map Oberkirchfeld. Wenn ich am Spiel anfang das Feld automatisch Dreschen lassen möchte, kommt immer die Meldung das dass Feld nicht generiert werden kann.

**Comment by Tensuko** [comment]
Da brauche ich mehr Infos.
Welches Feld, was für ein Drescher und wie genau versuchst du es?

**Comment by Gjardiunn** [comment]
New Holand CH7.70
Feld 1
Ich fahre an den Feldrand, öffne esc menue und da auf job erstellen. dann klicke ich auf CP Feldarbeit, feldposition und danach Zielposition.  Als letztes auf lkurs generieren
[log Kopie.txt](https://github.com/Courseplay/Courseplay_FS22/files/12310012/log.Kopie.txt)

**Reply by Tensuko** [reply]
Die Felder sind zu dicht beisammen.
Es kann sein das mehrere Felder markiert werden wie hier:
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/826c66ca-9ab8-4cd8-945c-600ded5ae732)

Versuch es so:
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/bc444cb6-dd1f-463c-b01d-43402c20113f)

Bei mir klappt es dann mit 7.3.0.0
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/d2d29e3b-6db5-4480-ab94-1bca350acab5)

**Comment by Gjardiunn** [comment]
danke
